tools/slack/external-collaboration.md

```diff
@@ -0,0 +1,72 @@
+---
+title: External collaboration through Slack
+---
+
+[_Back to Slack page_](../)
+
+Collaboration with people outside of TTS falls into one of two categories:
+
+- [Partners](#partners)
+- [The public](#the-public)
+
+We use different setups for each.
+
+## Partners
+
+Collaborators may include:
+
+- Vendors _under contract_ (not just a Terms of Service) with TTS
+- Vendors with GSA who are working on a project with TTS, for which we have an Memorandum of Understanding (MOU)
+- Partners at federal agencies whose projects are under an Interagency Agreement with TTS, or at state or local agencies whose projects are under an Intergovernmental Collaboration Act agreement with TTS.
+
+Note this does _not_ include:
+
+- The general public, for which we have [dedicated channels](#the-public)
+- Other feds who we are not working with directly
+
+Contractors' level of access will be determined by the Contracting Officer (CO). By default, **contractors who are "embedded" in TTS (working on TTS projects most or full-time) can be added as full members, while other collaborators should be added as single or multi-channel guests**, as appropriate.
+
+Additonal exceptions, including a small number of close TTS collaborators at GSA, may be added as full Slack members at the discretion of the Technology Director. Examples might include the Chief Information Officer or FAS Commissioner. Ask in [#admins-slack](https://gsa-tts.slack.com/messages/admins-slack) if you'd like to add someone to this category.
+
+### Dedicated channels
+
+You may want to invite partners, contractors, etc. to specific project channels to foster collaboration and asynchronous communication with the team. Projects often [create](https://get.slack.help/hc/en-us/articles/201402297-Creating-a-channel) specific channels that end with `-partners`.
+
+1. [Create a `<project>-partners` channel](https://get.slack.help/hc/en-us/articles/201402297-Creating-a-channel), if you don't want them in an existing channel.
+1. Add the partners to the channel, either:
+   - Through Slack Connect (preferred; see below)
+   - Individually, through the [user request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSfYQ-D82rIGwbCmwF3kAQERqczi5syVGq6GtmQNR6fhxRAA2Q/viewform)
+
+#### Slack Connect
+
+Slack offers [this mechanism](https://slack.com/help/articles/115004151203-Guide-to-sharing-channels-with-external-organizations) to connect channels between different workspaces. If our partner has their own Slack workspace, this is the preferred method for collaborating with them from TTS Slack, since:
```

Comment:
>this is the preferred method for collaborating with them from TTS Slack

we suggest using theirs instead of our Slack, since:

Reply:
Sorry, what do you mean?